### DISCUSSION TITLE
cefsharp proxy authentication required ?how to do it ?

### DISCUSSION THREAD

**Discussion by AtlantisDe** [cefsharp/CefSharp#3954] [General]
cefsharp proxy authentication required ?how to do it ?
- [RequestContextExtensions.cs](https://github.com/cefsharp/CefSharp/blob/master/CefSharp/RequestContextExtensions.cs)

---

`i'm trying to use proxy with Auth on cefsharp i tried this code and it's working with proxy without Auth only what should i do to set Auth .`

--- 

```c#
Cef.UIThreadTaskFactory.StartNew(delegate
                    {
                        string ip = "IP";
                        string port = "PORT";
                        var rc = chrome.GetBrowser().GetHost().RequestContext;
                        var dict = new Dictionary<string, object>();
                        dict.Add("mode", "fixed_servers");
                        dict.Add("server", "" + ip + ":" + port + "");
                        string error;
                        bool success = rc.SetPreference("proxy", dict, out error);
                    });
```

**Comment by amaitland** [comment]
In future please use discussions to ask questions questions. 

See https://github.com/cefsharp/CefSharp/wiki/General-Usage#proxy-authentication for proxy auth
